src/Data/Rational/Properties.agda

```diff
@@ -1638,6 +1638,10 @@ toℚᵘ-homo-∣-∣ (mkℚ -[1+ _ ] _ _) = *≡* refl
 ∣-p∣≡∣p∣ (mkℚ (+ zero) d-1 _) = refl
 ∣-p∣≡∣p∣ (mkℚ -[1+ n ] d-1 _) = refl
 
+∣p∣≡p⇒0≤p : ∀ {p} → ∣ p ∣ ≡ p → 0ℚ ≤ p
+∣p∣≡p⇒0≤p {p} ∣p∣≡p =
+  toℚᵘ-cancel-≤ (ℚᵘ.∣p∣≃p⇒0≤p (ℚᵘ.≃-trans (ℚᵘ.≃-sym (toℚᵘ-homo-∣-∣ p)) (ℚᵘ.≃-reflexive (cong toℚᵘ ∣p∣≡p))))
```

Comment:
Might be worth using the equational reasoning here after `toℚᵘ-cancel-≤ (ℚᵘ.∣p∣≃p⇒0≤p ...)` to make it a little bit more readable?

Reply:
Fixed